### DISCUSSION TITLE
[QA] EDIT

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#266] [Q&A]
edit

**Comment by NamgyungKim** [comment]
글이 수정되고 아무런 액션이 없습니다.
글이 수정된건지 모르겠어요~~

**Reply by Hyevvy** [reply]
요거는 확실히 말씀드릴 수 있군요 !! 
[수정페이지 서버통신 PR](https://github.com/prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung/pull/221)을 보면 alert가 뜨는 걸 볼 수 있습니당! 서버 오류가 나서 확인이 안 된거 같네욤 ..

**Comment by NamgyungKim** [comment]
예상기간을 4개월로 해놨었는데
edit버튼을 누르고 페이지를 보니 1개월 기본값으로 변경 되어있습니다.~

**Reply by Hyevvy** [reply]
이것도 서버문제로 인해 그런 거 같군요 ㅠ 속상하군요 ㅠ

**Reply by Hyevvy** [reply]
이건 제가 잘못했군요!!  버그 파겠습니당

**Reply by Hyevvy** [reply]
이 부분은 수정 완료 했습니다!! 
[해당 issue](https://github.com/prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung/issues/287)
